### PR TITLE
Introduction

### PR DESCRIPTION
**Pull request by xy2iii**
First step on https://github.com/time-rs/time/issues/327.

For writing this documentation I'm following the [four types of documentation](https://documentation.divio.com/introduction/) style, which separates tutorials, how-tos, references and deep explainations. Not sure what you think about this system, we will need all four, but I just did a simple tutorial as the first page for now. Existing pages fit as references and explainations.

This intro page:
- gives a short pitch of time
- includes a short tutorial to get started with basic functions
- gives a comparaison with chrono.

Let me know what you think.

### CODE REVIEW

**Comment by jhpratt** [comment]
I'll take a look at this when I get a chance. On the assumption it's something meaningful, I'll likely have no issue with it.

File: src/index.md [hunk 1]
```diff
@@ -1 +1,81 @@
-# What is Time?
+# Introduction
+
+`time` is a date and time library for Rust. It's:
```
Comment (jhpratt) [inline src/index.md:3]: Given that nothing comes immediately after "it's", it feels better to say "it is".

File: src/index.md [hunk 2]
```diff
@@ -1 +1,81 @@
...
+
+This short tutorial describes basic usage of `time`, to get operational quickly.
+
+1. **Install `time`**.  Add it to your `Cargo.toml`. We'll enable `macros`:
+```toml
+[dependencies]
+time = { version = "0.3", features = ["macros"] }
+```
+
+2. **Create dates and times.** We can create dates ([`Date`](https://docs.rs/time/latest/time/struct.Date.html)), dates with times ([`PrimitiveDateTime`](https://docs.rs/time/latest/time/struct.PrimitiveDateTime.html)) and date times with an UTC offset ([`OffsetDateTime`](https://docs.rs/time/latest/time/struct.OffsetDateTime.html)). A simple [`Time`](https://docs.rs/time/latest/time/struct.Time.html) is also available.
```
Comment (jhpratt) [inline src/index.md:26]: This line (and any others that are sufficiently long) should be wrapped to 100 characters.

File: src/index.md [hunk 3]
```diff
@@ -1 +1,81 @@
...
+
+Around 2019, the time crate, which was unmaintained since August 2016, was picked up for
+maintenance again. `time` has since been rewritten as of time 0.2, and is incompatible with
+the 0.1 version.
+
+Today:
+- `time` has been rewritten from 0.1,and is actively developed.
+- `chrono` depends on time 0.1, an old version unrelated with current `time`, and is actively developed as well.
+
+Since they are incompatible with each other, please choose the library that fits your needs.
```
Comment (jhpratt) [inline src/index.md:81]: How does a user know which library fits their needs?

This doesn't need to be addressed in this PR, but it's something to keep in mind :slightly_smiling_face:

**Review by jhpratt — COMMENTED** [review]
Minimal changes for readability. Otherwise LGTM.

File: src/index.md (hunk 3 above)
Comment (xy2iii) [reply src/index.md:97]: I guess we'll find out by askng them :)

**Pull request merged**